server/src/recipe/handler/recipe.handler.ts

```diff
@@ -70,7 +81,9 @@ export class RecipeHandler {
   }
 
   public update = async (req: Request, res: Response) => {
-    const recipe: Recipe = new Recipe().clone(req.body as Recipe);
+    let recipe: Recipe = new Recipe().clone(req.body as Recipe);
```

Comment:
if you provide a deleteImage Function into the recipe model object you could use ```const``` again.

Reply:
Nice tip!